core/server/web/api/canary/admin/routes.js

```diff
@@ -109,9 +108,6 @@ module.exports = function apiRoutes() {
     // ## Roles
     router.get('/roles/', mw.authAdminApi, http(apiCanary.roles.browse));
 
-    // ## Clients
-    router.get('/clients/slug/:slug', api.http(api.clients.read));
```

Comment:
Just need to make sure no other system like Pro is using this one.

Reply:
@gargol Checked with pro and its not being used anywhere, so think its safe to cleanup 😄